spec/lib/api/paging/index_paginator_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+describe Api::Paging::IndexPaginator do
+  before do
+    (1..7).each do |i|
+      public = !(i == 1 || i == 6)
```

Comment:
Style/MultipleComparison: Avoid comparing a variable with multiple items in a conditional, use `Array#include?` instead.